masterlist.yaml

```
@@ -22646,3 +22646,11 @@ plugins:
     clean:
       - crc: 0xBD12E363
         util: 'SSEEdit v4.0.4'
+
+  - name: 'WorldEaterBeater.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/60778' ]
```

As Sabre said, please add a trailing slash for consistency with the rest of the masterlist.

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/60778/' ]
```